src/test/java/core/basesyntax/service/impl/ReaderCsvTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class ReaderCsvTest {
+
+    @TempDir
+    private Path tempDir;
+
+    @Test
+    void readData_ok() throws IOException {
+        Path csvPath = tempDir.resolve("test.csv");
```

Comment:
Is it possible to use some constants to add additional context for some strings?